invenio_app_ils/acquisition/ext.py

```diff
@@ -33,11 +33,11 @@ def record_class_by_pid_type(self, pid_type):
         endpoints = self.app.config.get('RECORDS_REST_ENDPOINTS', [])
         return endpoints[pid_type]['record_class']
 
-    def search_by_pid_type(self, pid_type):
+    def search_class_by_pid_type(self, pid_type):
         endpoints = self.app.config.get("RECORDS_REST_ENDPOINTS", [])
         return endpoints[pid_type]["search_class"]
 
-    def indexer_by_pid_type(self, pid_type):
+    def indexer_class_by_pid_type(self, pid_type):
```

Comment:
just make sure that you will always instantiate the cls when using it: in the main `ext.py`, we return an instance and not a class if I am not mistaken. Check the usage of it.